pkg/install/storage.go

```diff
@@ -254,17 +235,41 @@ func (i *Installer) installStorage(ctx context.Context, installConfig *installco
 		})
 	}
 
-	err = i.deployARMTemplate(ctx, resourceGroup, "storage", t, nil)
-	if err != nil {
-		return err
+	return i.deployARMTemplate(ctx, rgName, "storage", t, nil)
+}
+
+func (i *Installer) ensureGraphExists(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
+	// If graph has already been created, don't create a new one
+	_, err := i.loadGraph(ctx)
+	if err == nil {
+		i.log.Print("graph already exists")
+		return nil
 	}
 
-	// the graph is quite big so we store it in a storage account instead of in cosmosdb
-	err = i.saveGraph(ctx, g)
-	if err != nil {
-		return err
+	clusterID := &installconfig.ClusterID{
```

Comment:
_Tip: Ctrl+F and rapidly looping through results for this line makes it easy to see how this code was moved but unaltered._

![image](https://user-images.githubusercontent.com/24868312/78375311-a1e87180-759a-11ea-81cc-e47c194d705e.png)